test/hotspot/jtreg/runtime/CompressedOops/CompressedClassPointers.java

```diff
@@ -318,6 +319,12 @@ public static void sharingTestNoCoop() throws Exception {
     }
 
     public static void main(String[] args) throws Exception {
+        // Testing compressed class pointers without compressed oops.
+        // This is only possible if the platform supports it. Notably,
+        // on macOS, when compressed oops is disabled and the heap is
+        // given an arbitrary address, that address occasionally collides
+        // with where we would ideally have placed the compressed class
+        // space. Therefore, macOS is omitted for now.
```

Comment:
This really belong as an `@comment` in the test comment block now - but somewhat shorter.

Reply:
Omit the "for now" in the comment also since I don't think we're going to start supporting macosx.